### PR TITLE
Fixed golint issue in cas_template_key.go

### PR DESCRIPTION
**Pull request by RajuKalidindi**
Signed-off-by: RajuKalidindi <krkraju997@gmail.com>

**What this PR does / why we need it**: Fixed golint issue in cas_template_key.go

**Which issue this PR fixes**: fixes [#1961](https://github.com/openebs/openebs/issues/1961)

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by princerachit** [comment]
@RajuKalidindi please do a make test and fix the test files.

**Comment by sonasingh46** [comment]
Similarly for other places. ^^^

File: pkg/apis/openebs.io/v1alpha1/cas_template_keys.go [hunk 1]
```diff
@@ -16,14 +16,15 @@ limitations under the License.
 
 package v1alpha1
 
+// CasKey ...
```
Comment (princerachit) [inline pkg/apis/openebs.io/v1alpha1/cas_template_keys.go]: CasKey is used for all cas specific annotation keys

File: pkg/apis/openebs.io/v1alpha1/cas_template_keys.go [hunk 2]
```diff
@@ -109,7 +109,8 @@ const (
 	// NOTE:
 	//  The corresponding value will be accessed as
 	// {{ .Storagepool.owner }}
-	OwnerCTP    StoragePoolTLPProperty = "owner"
+	OwnerCTP StoragePoolTLPProperty = "owner"
+	// DiskListCTP ...
```
Comment (princerachit) [inline pkg/apis/openebs.io/v1alpha1/cas_template_keys.go]: indicates the list of disks

File: pkg/apis/openebs.io/v1alpha1/cas_template_keys.go [hunk 3]
```diff
@@ -187,6 +188,7 @@ const (
 )
 
 const (
+	// TaskIdentityPrefix ...
```
Comment (princerachit) [inline pkg/apis/openebs.io/v1alpha1/cas_template_keys.go]: is the prefix used for all TaskIdentity

**Comment by RajuKalidindi** [comment]
@princerachit  Thanks for helping out with the comments

**Pull request merged**